bool active
string target
string obsmode
string obs_script
string obs_file
int32 num_on
int32 num_seq
float64 xgrid
float64 ygrid
float64 exposure_hot
float64 exposure_off
float64 exposure_on
string scan_direction

int32 current_num
string current_position

string from_node
float64 timestamp